### ISSUE TITLE
Questions about SceneFlow's pre-training model

### ISSUE THREAD

**Issue by lsyl20** [Windsrain/Selective-Stereo#22]
Thank you for your wonderful work!

1. When I tested the Selective-RAFT model SceneFlow's pre-trained model, I found differences in the 1px results. Here are my test results: `Validation Scene Flow: EPE: 0.469462, 1px: 5.422181`
2. Also, I would like to know how the accuracy of Baseline (RAFT-Stereo) mentioned in Table 1 of the paper was obtained.

Thank you and look forward to your reply.

**Comment by Windsrain** [comment]
1. Depending on different devices, there may be some deviations. You can record the indicators according to your testing records.
2. The baseline is our re-trained RAFT-Stereo, we set the max disparity of sequence loss to 192, the same as IGEV-Stereo.

**Comment by lsyl20** [comment]
Thank you for your prompt response, which gave me a lot of inspiration.

The accuracy of the RAFT-Stereo model I retrained using the default settings is lower than what you mentioned. I'm not sure if this is due to the loss settings. The default settings for the sequence loss are as follows:`def sequence_loss(flow_preds, flow_gt, valid, loss_gamma=0.9, max_flow=700):`

**Comment by lsyl20** [comment]
> 1. Depending on different devices, there may be some deviations. You can record the indicators according to your testing records.
> 2. The baseline is our re-trained RAFT-Stereo, we set the max disparity of sequence loss to 192, the same as IGEV-Stereo.

Thank you for your prompt response, which gave me a lot of inspiration.

The accuracy of the RAFT-Stereo model I retrained using the default settings is lower than what you mentioned. I'm not sure if this is due to the loss settings. The default settings for the sequence loss are as follows:`def sequence_loss(flow_preds, flow_gt, valid, loss_gamma=0.9, max_flow=700):`

**Comment by Windsrain** [comment]
change the max_flow=700 to max_flow=192.

**Comment by lsyl20** [comment]
> * Depending on different devices, there may be some deviations. You can record the indicators according to your testing records.

thanks

**Comment by lsyl20** [comment]
Sorry to bother you again. May I ask what the training curves of EPE on the training set and on the validation set look like during training?

**Comment by Windsrain** [comment]
The EPE on the training set will initially decrease rapidly and then fluctuate within a certain range. The EPE on the test set will continue to decrease.

**Comment by lsyl20** [comment]
> The EPE on the training set will initially decrease rapidly and then fluctuate within a certain range. The EPE on the test set will continue to decrease.

Is the EPE on the training set higher than that on the validation set?

**Comment by Windsrain** [comment]
This cannot be compared. The EPE on the training set is for samples with data augmentation after crop, and cannot be compared with the test set.

**Comment by lsyl20** [comment]
> This cannot be compared. The EPE on the training set is for samples with data augmentation after crop, and cannot be compared with the test set.

Thank you for your reply. It has resolved my confusion.
